enhancements/sdk-generate-commands.md

```diff
@@ -0,0 +1,353 @@
+---
+title: README describing new command workflows for OLM enabled operators
+authors:
+  - "@varshaprasad96"
+  - "@estroz"
+reviewers:
+  - "@jlanford"
+  - "@dmesser"
+  - "@jmrodri"
+approvers:
+  - "@jlanford"
+  - "@dmesser"
+  - "@jmrodri"
+creation-date: 2020-04-09
+last-updated: 2020-06-02
+status: implementable
+see-also:
+  - https://github.com/operator-framework/operator-sdk/pull/2860
+---
+
+# New `generate` command workflows for OLM enabled operators
```

Comment:
This should be the Title of the proposal. I usually make it match the filename but in mixed case. I.e. if my title was `sdk-generate-commands` then I'd make it `# SDK Generate Commands`.